inference-engine/tests/functional/plugin/conformance/test_runner/api_conformance_runner/include/api_conformance_helpers.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "conformance.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+// TODO: fix namespaces
+using namespace ConformanceTests;
+
+namespace ov {
+namespace test {
+namespace conformance {
+
+inline const std::string getPluginLibNameByDevice(const std::string& deviceName) {
+    const std::map<std::string, std::string> devices{
+            { "AUTO", "AutoPlugin" },
+            { "HDDL", "HDDLPlugin" },
+            { "VPUX", "VPUXPlugin" },
+            { "AUTO", "MultiDevicePlugin" },
+            { "CPU", "MKLDNNPlugin" },
+            { "GNA", "GNAPlugin" },
+            { "GPU", "clDNNPlugin" },
+            { "HETERO", "HeteroPlugin" },
+            { "MULTI", "MultiDevicePlugin" },
+            { "MYRIAD", "myriadPlugin" },
+            { "TEMPLATE", "templatePlugin" },
+    };
+    if (devices.find(deviceName) == devices.end()) {
+        throw std::runtime_error("Incorrect device name");
+    }
+    return devices.at(deviceName);
```

Comment:
is it used? for what?

Reply:
![image](https://user-images.githubusercontent.com/22008923/141088369-2af4ebda-a6fe-407c-855b-632f3e9c2546.png)

Reply:
![image](https://user-images.githubusercontent.com/22008923/141091306-cb56c1a2-e4bf-474e-afbd-601eb0acebb0.png)